src/ext/ca/serverca/scasched/scauser.cpp

```diff
@@ -540,6 +540,8 @@ HRESULT ScaUserExecute(
             ueUserExists = USER_EXISTS_INDETERMINATE;
             hr = HRESULT_FROM_WIN32(er);
             WcaLog(LOGMSG_VERBOSE, "Failed to check existence of domain: %ls, user: %ls (error code 0x%x) - continuing", wzDomain, psu->wzName, hr);
+			hr = S_OK;
+			er = ERROR_SUCCESS;
```

Comment:
No, that's just a Notepad++ thing
The change is resetting er and hr to success values. 
Otherwise the error code is dragged on to the return when the function has nothing more to do (subsequent 'if' statements are all false)